packages/@storefront/core/CHANGELOG.md

```diff
@@ -4,6 +4,11 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [Unreleased] [patch]
+### Added
+- SF-1163: Add hardcoded `GBI_EVENT` to the metadata of all tracking events.
+  - `GBI_EVENT` indicates to our tracking service that an event is of gbi origin.
```

Comment:
Show the signature of `GBI_EVENT`, eg `{ key: 'gbi', value: true }` or whatever it is, that way a user can more easily see where this was added if they don't know about `GBI_EVENT`

Reply:
Switch to this:
```
const GBI = 'gbi';
const GBI_EXPERIENCE = 'gbi_experience';
const GBI_METADATA = [    {
      key: GBI
      value: 'true'
    }, 
  {
    key: GBI_EXPERIENCE,
    value: 'storefront'
  }]
```

Reply:
(reference the correct array GBI_METADATA, show the full thing). Maybe also explain that if an implementation attempts to provide its own values, storefront will remove them and replace them with the correct values for a storefront project

Reply:
Done, resolving conversation.